src/App.tsx

```diff
@@ -22,7 +27,43 @@ function App() {
 
   return (
     <>
-      <div>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" color="inherit" component="div">
+            Invoices
+          </Typography>
+
+          <MenuItem>
+            <Typography textAlign="center">New Invoice</Typography>
+          </MenuItem>
+        </Toolbar>
+      </AppBar>
+
+      <InvoiceList></InvoiceList>
+
+      <Invoice></Invoice>
+
+      {/* Pytanie: dlaczego jak daje m={2} dodaje mi poziomego scrolla */}
```

Comment:
If you inspect this container you will see that it has `width: calc(100% + [n]px)` where n is your margin. This is top level grid container so by default it expands to the full width, if you add any margin, it will exceed 100% causing horizontal scroll. We can discuss at friday ;)